lib/Mojolicious/Static.pm

```diff
@@ -71,7 +73,9 @@ sub is_fresh {
 
   # If-None-Match
   $etag //= $res_headers->etag // '';
-  return undef if $match && !grep { trim($_) eq $etag } split ',', $match;
+  return undef
+    if $match && !grep { my $m = trim($_); $m eq $etag || "W/$m" eq $etag }
+    split ',', $match;
```

Comment:
I'm sure this fits on two lines too.

Reply:
I ran the file through perltidy (I had it originally on one line) and this is what it gave me.

Reply:
I mean with a better implementation.

Reply:
oh I see. ok I'll have a whirl at that.

Reply:
```perl
  my @match = map { trim($_) } split ',', $match;
  return undef if $match && !grep { $_ eq $etag || "W/$_" eq $etag } @match;
```